python/mathsys/main/builder.py

```
@@ -17,15 +17,18 @@ class Builder:
+        "web": "wasm32-unknown-unknown"
+    }
     target: str
     debug: bool
-    modules: list[str]
     location: str
     # CLASS -> INIT
     def __init__(self, data: bytes, target: str) -> None:
-        self.modules = ["all"]
         self.debug = False
         self.data = data
+        if target not in self.targets: raise ValueError(f"'{target}' is not a supported target.")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)